keps/sig-node/1287-in-place-update-pod-resources/README.md

```diff
@@ -277,65 +269,73 @@ the `/resize` subresource:
-`Pod.Spec.Containers[i].Resources.Requests` field differs from the
-`Pod.Status.ContainerStatuses[i].Resources` field, this new field explains why.
-
-This field can be set to one of the following values:
-* `Proposed` - the proposed resize (in Spec...Resources) has not been accepted or
-  rejected yet. Desired resources != Allocated resources.
-* `InProgress` - the proposed resize has been accepted and is being actuated. A new resize request
-  will reset the status to `Proposed`. Desired resources == Allocated resources != Actual resources.
+Resize status will be tracked via 2 new pod conditions: `PodResizePending` and `PodResizing`.
+
+**PodResizePending** will track states where the spec has been resized, but the Kubelet has not yet
+allocated the resources. There are two reasons associated with this condition:
+
 * `Deferred` - the proposed resize is feasible in theory (it fits on this node)
-  but is not possible right now; it will be re-evaluated.
-  Desired resources != Allocated resources.
-* `Infeasible` - the proposed resize is not feasible and is rejected; it will not
-  be re-evaluated. Desired resources != Allocated resources.
-* (no value) - there is no proposed resize.
-  Desired resources == Allocated resources == Acutal resources.
+  but is not possible right now; it will be regularly reevaluated.
+* `Infeasible` - the proposed resize is not feasible and is rejected; it may not
+  be re-evaluated.
+
+In either case, the condition's `message` will include details of why the resize has not been
+admitted. `lastTransitionTime` will be populated with the time the condition was added. `status`
+will always be `True` when the condition is present - if there is no longer a pending resized
+(either the resize was allocated or reverted), the condition will be removed.
 
-Any time the apiserver observes a proposed resize (a modification of a
-`Spec...Resources` field), it will automatically set this field to `Proposed`.
+**PodResizing** will track in-progress resizes, and should be present whenever allocated resources
+!= acknowledged resources (see [Resource States](#resource-states)). For successful synchronous
+resizes, this condition should be short lived, and `reason` and `message` will be left blank. If an
+error occurs while actuating the resize, the `reason` will be set to `Error`, and `message` will be
+populated with the error message. In the future, this condition will also be used for long-running
+resizing behaviors (see [Memory Limit Decreases](#memory-limit-decreases)).
 
-To make this field future-safe, consumers should assume that any unknown value
-means the same as `Deferred`.
+Note that it is possible for both conditions to be present at the same time, for example if an error
+is encountered while actuating a resize and a new resize comes in that gets deferred.
+
+Prior to v1.33, the resize status was tracked by a dedicated `Pod.Status.Resize` field. This field
```

Comment:
Please update the [`API Changes` section above](https://github.com/kubernetes/enhancements/pull/5089/files#diff-14542847beb0f0fd767db1aff1316f8569a968385e2bb89567c4cc0af1ae5942R223-R224) to reflect this deprecation and move to PodConditions.